index.html

```diff
@@ -39,11 +39,11 @@ <h1 class="primary-heading">Nurgul Kereikhan</h1>
           </p>
           <button class="cta-btn">Start collaboration</button>
         </div>
-        <img class="decoration-desktop" src="./assets/desktop/image-geometry_1.png" alt="geometry-1">
+        <img class="decoration-desktop" src="./assets/desktop/image-geometry_1.png" alt="Its decoration background geometry image">
```

Comment:
Please note that the decoration images don't need an alt text as described [here](https://www.w3.org/WAI/test-evaluate/preliminary/#images) 😃

![image](https://user-images.githubusercontent.com/73714615/188655310-bb41d253-4543-4550-92a2-a8484ddb946a.png)